### PR TITLE
[Collection] add test to CollectionApp

### PR DESCRIPTION
**Pull request by l2succes**
Related: https://artsyproduct.atlassian.net/browse/GROW-949

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit e495e82c0c3312518f732563f504308784e307db

https://deploy-preview-1426--artsy-reaction.netlify.com

File: src/Apps/Collect/__tests__/CollectionApp.test.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+  beforeEach(() => {
+    filterState = new FilterState({
+      tracking: {
+        trackEvent: jest.fn(),
+      },
+    })
+  })
+
+  it("renders a relay tree correctly", async () => {
+    const tree = await renderUntil(
```
Comment (alloy) [inline src/Apps/Collect/__tests__/CollectionApp.test.tsx]: Was there a reason you didn’t use `renderRelayTree`?

Comment (l2succes) [reply src/Apps/Collect/__tests__/CollectionApp.test.tsx]: It didn't work for me when I first tried but I may have done something wrong. so I ended up testing different conditionals to stop the render loop. 
Also I wanted to understand the underlying parts of it but we can always update it after

Comment (alloy) [reply src/Apps/Collect/__tests__/CollectionApp.test.tsx]: Great, understanding the parts is always a good thing 👍 Let’s connect on Monday about why it didn’t work for you.

Comment (l2succes) [reply src/Apps/Collect/__tests__/CollectionApp.test.tsx]: 👍sounds good

Comment (alloy) [reply src/Apps/Collect/__tests__/CollectionApp.test.tsx]: Is there by any chance a nested QueryRenderer that isn’t using renderWithLoadProgress?

Comment (l2succes) [reply src/Apps/Collect/__tests__/CollectionApp.test.tsx]: I don't think so, there's only one root QueryRenderer. I'll try switching it again to see what happens

Comment (alloy) [reply src/Apps/Collect/__tests__/CollectionApp.test.tsx]: Or if there’s any other special work that happens that may have been deferred until a later render pass? In which case you should use the `renderUntil` parameter for `renderRelayTree` (which appears to still be called `until` in the doc) https://github.com/artsy/reaction/blob/bdc26827d58b78b5c13a6603d6b8768269c67c4e/src/DevTools/renderRelayTree.tsx#L25-L27

**Pull request merged**